range_queries/sparse_table.cpp

```diff
@@ -70,24 +77,23 @@ std::vector<std::vector<T> > buildTable(int n, const std::vector<T>& A, const st
  * @param table sparse table data structure for the input array
  * @return minimum value for the [beg, end] range for the input array
  */
-template<typename T>
-int getMinimum(int beg, int end, const std::vector<T>& logs, const std::vector<std::vector<T> >& table) {
+int getMinimum(int beg, int end, const std::vector<int>& logs, const std::vector<std::vector<int> >& table) {
```

Comment:
the change made here is correct. the previous code here using the template was correct.

Reply:
Done.